src/test/acceptance/features/enter-name-steps.js

```diff
@@ -1,26 +1,17 @@
-const { Before, Given, When, Then, After } = require('cucumber')
+const { Given, When, Then } = require('cucumber')
 const { expect, assert } = require('chai')
 
-const EnterName = require('../../common/page/enter-name')
-const Overview = require('../../common/page/overview')
-const Confirmation = require('../../common/page/confirmation')
-const DriverManager = require('../../common/driver/driver-manager')
+const pages = require('./pages')
 const { BASE_URL } = require('../constants')
 
-const driverManager = new DriverManager()
-let enterName
-let overview
-let confirmation
-let driver
-
 const LONG_NAME = 'This name is way too long' +
   'This name is way too long' +
   'This name is way too long' +
   'This name is way too long' + // 100
   'This name is way too long' +
   'This name is way too long' +
   'This name is way too long' +
-  'This name is way too long' + // 200
+  'This name is way too long' + // 200a
```

Comment:
200a? :)

Reply:
No idea, I've corrected it on the smoke test branch that I'm currently working on.